packages/transaction-controller/src/TransactionController.ts

```diff
@@ -701,9 +711,11 @@ export class TransactionController extends BaseController<
 
   private readonly beforeCheckPendingTransaction: (
     transactionMeta: TransactionMeta,
-  ) => boolean;
+  ) => Promise<boolean>;
 
-  private readonly beforePublish: (transactionMeta: TransactionMeta) => boolean;
+  private readonly beforePublish: (
+    transactionMeta: TransactionMeta,
+  ) => Promise<boolean>;
```

Comment:
The description mentions `Before publish hook now receives raw transaction`? Was that removed?

Reply:
Well spotted, thank you